Issue4～6の修正

masaru-kunimatsu opened the pull request.

masaru-kunimatsu committed:
Issue4

masaru-kunimatsu committed:
Issue5

masaru-kunimatsu committed:
Issue6

masaru-kunimatsu commented:
プルリクエスト作成時にキャプチャ動画のURLを貼り忘れてしまいました。こちらが動画になります。https://gyazo.com/e516af914f2bec0fb56114715a1e3fb8

TAKADA-yuma commented:
レビュー依頼ありがとうございます！

プルリクエストの内容を拝見したところ、現状こちらのプルリクエストにて、
コンフリクトが起きておりレビューが行えない状態です。

そのため、まずはコンフリクトの解消を行っていただけますでしょうか。

ブラウザエディタでコンフリクトを解消できますので、
「Resolve conflicts」をクリックしましょう。

もしご不明点がありましたら、コネクトにてご質問いただけると幸いです。

Kentozzz commented on app/controllers/calendars_controller.rb:
```diff
@@ -34,7 +34,15 @@ def get_week
+
+
+      wday_num = (@todays_date+x).wday
+      if #「wday_numが7以上の場合」という条件式
```
```
if #「wday_numが7以上の場合」という条件式
```
if の後に条件式を書きましょう！
「もしwdaynumが7以上なら」と言う条件を作成してみてください。

Kentozzz commented on app/controllers/calendars_controller.rb:
```diff
@@ -34,7 +34,15 @@ def get_week
+        wday_num = wday_num -7
+      end
+
+      days = { :month => (@todays_date + x).month, :date => (@todays_date+x).day, :wday => wdays[wday_num] ,:plans => today_plans}
```
```
days = { :month => (@todays_date + x).month, :date => (@todays_date+x).day, :wday => wdays[wday_num] ,:plans => today_plans}
```
・ハッシュロケットをシンボル型に修正しましょう！

Kentozzz commented on app/views/calendars/index.html.erb:
```diff
@@ -9,7 +9,7 @@
       <div class='item'>
         <div class='date'>
-          <% day[:month] %>/<% day[:date] %>
+          <%= day[:month] %>/<%= day[:date] %>/<%= day[:wday] %>
```
```
<%= day[:month] %>/<%= day[:date] %>/<%= day[:wday] %>
```
日付と曜日の間に/が含まれています！
見本アプリと表示を合わせましょう！

Kentozzz reviewed (commented):
レビュー依頼ありがとうございます！
コメントしましたので、ご確認をお願いいたします！

コメントに合わせて修正した後は、コミット→プッシュしてレビュー再依頼をお願いいたします。

masaru-kunimatsu committed:
Issue4～6　再度修正

頂いたメッセージをもとに再度修正しました。ご確認お願いします。https://gyazo.com/91455d07c08ff9b841bbffb58b013092

masaru-kunimatsu committed:
Merge branch 'master' into second_training

OhyaYouki reviewed (commented):
レビュー依頼ありがとうございます！

**LGTM**とさせていただきます！

これでissue1~6全てのissueが解消されました。お疲れ様でした！
忘れずにこのブランチをMergeしていただき、ローカルのMasterにPullをお願いいたします。

次のカリキュラムも頑張っていきましょう！応援しております！
